portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro

```diff
@@ -50,9 +50,15 @@ definition {
 
 		FormPortletNavigator.gotoSelectForm();
 
+		if (isSet(formTitle)) {
```

Comment:
If I don't add this condition, ${formTitle} will always be "Created Form Name" because this variable is created in this macro file. So I needed to add this to cases where the test name is different

Reply:
Hey @davisaints, what if we just update the current and new tests to call formName? Given that there is only one test that uses it, it should be easy. The macro can be updated to also set formTitle to formName. Let me know what you think! Thanks

Reply:
It is better!